aspnetcore/fundamentals/servers/kestrel/sample/Properties/launchSettings.json

```diff
@@ -0,0 +1,28 @@
+{
+  "iisSettings": {
+    "windowsAuthentication": false,
+    "anonymousAuthentication": true,
+    "iisExpress": {
+      "applicationUrl": "http://localhost:8096/",
+      "sslPort": 0
+    }
+  },
+  "profiles": {
+    "IIS Express": {
+      "commandName": "IISExpress",
+      "launchBrowser": true,
+      "environmentVariables": {
+        "ASPNETCORE_ENVIRONMENT": "Development"
+      }
+    },
+    "KestrelDemo": {
+      "commandName": "Project",
+      "commandLineArgs": "",
+      "launchBrowser": true,
+      "launchUrl": "http://localhost:5000",
```

Comment:
You're binding to "http://127.0.0.1:0" but launching "http://localhost:5000". The dynamic port 0 trick is primarily used for testing, show a static binding for the sample.